pyproject.toml

```diff
@@ -56,3 +56,7 @@ skips = ["B608"]
 
 [tool.codespell]
 ignore-words=".codespell-ignore"
+
+[tool.pytest.ini_options]
+filterwarnings = ["ignore::DeprecationWarning:pkg_resources",
+    "ignore::DeprecationWarning:cytominer_database"]
```

Comment:
```suggestion
filterwarnings = [
  "ignore::DeprecationWarning:pkg_resources",
  "ignore::DeprecationWarning:cytominer_database",
]
```

Reply:
Great, thanks! Applying this. Wondering about TOML formatters and pre-commit as a result of your suggestion ... 🤔